src/go/rpk/pkg/cli/cluster/storage/cancel-mount.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2024 Redpanda Data, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package storage
+
+import (
+	"strconv"
+
+	"github.com/redpanda-data/common-go/rpadmin"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/adminapi"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/config"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/out"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+)
+
+func newMountCancel(fs afero.Fs, p *config.Params) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "cancel-mount [MIGRATION ID]",
+		Short: "Cancels a mount/unmount operation on a topic.",
+		Long:  `Cancels a mount/unmount operation on a topic.`,
```

Comment:
Is worth explaining how to get the `MIGRATION-ID`

Reply:
I emit it every time a mount or unmount operation starts but I see your point. I should probably explain that it gets emitted + that it is available from list.